src/app/containers/StoryPromo/index.jsx

```diff
@@ -123,6 +131,10 @@ StoryPromoImage.defaultProps = {
   imageComponent: undefined,
 };
 
+const StyledPromoTimeStamp = styled(PromoTimestamp)`
+  ${({ promoType }) => promoType === 'onwardJourneys' && `color: ${C_GREY_6};`}
+`;
+
```

Comment:
Just for awareness as we're going down the compound component approach anyway, there might be a way to prevent prop drilling by targeting nested styled components https://emotion.sh/docs/styled#targeting-another-emotion-component. I've not tested it, it might not work, but I think this is what you can do following the documentation.

So at this container level you would export this styled component. Then at the top level you would import `StyledPromoTimeStamp` and override the component styling there.

src/app/containers/StoryPromo/index.jsx
```jsx
...
export const StyledPromoTimeStamp = styled(PromoTimestamp);
...
export const StoryPromoContainer = ({...
```

src/app/containers/CpsTopStories/index.jsx
```jsx
import {StoryPromoContainer, StyledPromoTimeStamp} from '../StoryPromo';
...
const CpsTopStoriesPromo = styled(StoryPromoContainer)`
   ${StyledPromoTimeStamp} {
      color: ${C_GREY_6};
   }
`;

```